### ISSUE TITLE
False positive alarm on unused class when using scss & parent selector

### ISSUE THREAD

**Issue by chepash** [atfzl/eslint-plugin-css-modules#96]
The ESLint warning is stating that the classes carouselListActive and carouselItemActive are detected as unused in the SubMenuCarousel.module.scss file. But they are not exist.

![image](https://github.com/atfzl/eslint-plugin-css-modules/assets/100382249/42ebef96-b7e1-45c4-a2ab-63b01a1e46de)

![image](https://github.com/atfzl/eslint-plugin-css-modules/assets/100382249/8433b3d1-e9ed-4a6c-9b92-0b01b6cfd25d)

TSX
```
import cn from 'classnames';
import { FC } from 'react';
import { NavLink } from 'react-router-dom';
import s from './SubMenuCarousel.module.scss';

export interface ISubMenuCarouselProps {
  className?: string;
}

export const SubMenuCarousel: FC<ISubMenuCarouselProps> = ({ className = '' }) => {
  return (
    <nav className={cn(s.carousel, className)}>
      <ul className={cn(s.carouselList)}>
        <li className={s.carouselItem}>
          <NavLink
            to="/"
            className={({ isActive }) => cn(s.carouselLink, { [s.carouselLinkActive]: isActive })}
          />
        </li>
      </ul>
    </nav>
  );
};
```

SCSS
```
.carousel {
  margin: 0;

  &List {
    margin: 0;
  }

  &Item {
    margin: 0;
  }

  &Link {
    margin: 0;

    &Active {
      text-decoration: underline;
    }
  }
}
```
